ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResourceTest.java

```diff
@@ -369,6 +369,25 @@ public void shouldRateLimit() {
 
   @Test
   public void queryLoggerShouldReceiveStatementsWhenHandleKsqlStatement() {
+    when(mockStatementParser.<Query>parseSingleStatement(PUSH_QUERY_STRING))
+        .thenReturn(query);
+    try (MockedStatic<QueryLogger> logger = Mockito.mockStatic(QueryLogger.class)) {
+      testResource.streamQuery(
+          securityContext,
+          new KsqlRequest(PUSH_QUERY_STRING, Collections.emptyMap(), Collections.emptyMap(), null),
+          new CompletableFuture<>(),
+          Optional.empty(),
+          KsqlMediaType.LATEST_FORMAT,
+          new MetricsCallbackHolder(),
+          context);
+
+      logger.verify(() -> QueryLogger.info("Transient query created",
+          PUSH_QUERY_STRING), times(1));
+    }
+  }
+
+  @Test
+  public void queryLoggerShouldNotReceiveStatementsWhenHandleKsqlStatement() {
```

Comment:
```suggestion
  public void queryLoggerShouldNotReceiveStatementsWhenHandlePullQuery() {
```
and rename the above test as well.

Reply:
oops... thanks!